### PR TITLE
SINGA -475 add Div operator

### PR DESCRIPTION
**Pull request by pinpom**

### CODE REVIEW

**Comment by chrishkchris** [comment]
This is a replacement of the previous Div operator PR.
All check is done (using docker container test environment). 
It is ready for merge.

**Comment by chrishkchris** [comment]
There are conflicts and there are two solutions (same as what happened in log operator):
1. merge this branch with the latest master
2. checkout a new branch from the latest master. Then copy the added code into the autograd.py and test_operations.py; send a new PR.

File: test/python/test_operation.py [hunk 1]
```diff
@@ -791,69 +791,23 @@ def test_Sub_cpu(self):
...
-        result = autograd.sub(x0, x1)
-        dx0, dx1 = result.creator.backward(dy.data)
-        DX0 = np.multiply(DY, 1.0)
-        DX1 = np.multiply(DY, -1.0)
-
-        np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT, decimal=5)
-        np.testing.assert_array_almost_equal(tensor.to_numpy(tensor.from_raw_tensor(dx0)), DX0, decimal=5)
-        np.testing.assert_array_almost_equal(tensor.to_numpy(tensor.from_raw_tensor(dx1)), DX1, decimal=5)
-        
-    def test_Pow_cpu(self):
```
Comment (nudles) [inline test/python/test_operation.py]: did you delete this function and others?
may due to the mis-use of git.

Comment (pinpom) [reply test/python/test_operation.py]: i guess i misused git command so some functions were accidently deleted. I'll create a new branch from latest master branch and create new PR for this operator

**Pull request closed without merging**